src/main/java/com/minecraftabnormals/buzzier_bees/core/other/BBCompat.java

```diff
@@ -10,10 +13,66 @@
 import net.minecraft.block.DispenserBlock;
 import net.minecraft.client.renderer.RenderType;
 import net.minecraft.client.renderer.RenderTypeLookup;
+import net.minecraft.dispenser.DefaultDispenseItemBehavior;
+import net.minecraft.dispenser.IBlockSource;
+import net.minecraft.entity.CreatureEntity;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.monster.EndermiteEntity;
+import net.minecraft.entity.monster.SilverfishEntity;
+import net.minecraft.entity.passive.BeeEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
 import net.minecraft.loot.LootTables;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.state.properties.BlockStateProperties;
+import net.minecraft.tileentity.DispenserTileEntity;
 import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvents;
+
+import java.util.function.Function;
 
 public class BBCompat {
+	public static final ImmutableMap<EntityType<?>, Function<Entity, ItemStack>> ENTITY_TYPE_TO_BOTTLE_MAP = ImmutableMap.of(
+			EntityType.BEE, (Entity entity) -> {
+				if (entity instanceof BeeEntity) {
+					BeeEntity bee = (BeeEntity) entity;
+					ItemStack bottle = new ItemStack(BBItems.BOTTLE_OF_BEE.get());
+					CompoundNBT tag = bottle.getOrCreateTag();
+					tag.putBoolean("HasNectar", bee.hasNectar());
+					tag.putBoolean("HasStung", bee.hasStung());
+					tag.putInt("AngerTime", bee.getAngerTime());
+					if (bee.getAngerTarget() != null) tag.putUniqueId("AngryAt", bee.getAngerTarget());
+					tag.putInt("Age", bee.getGrowingAge());
+					tag.putFloat("Health", bee.getHealth());
+					if (bee.hasCustomName()) {
+						bottle.setDisplayName(bee.getCustomName());
+					}
+				return bottle;
```

Comment:
This `return` line is not indented properly.

Reply:
Fixed